web/api/tests/routes/test_run.py

```diff
@@ -64,26 +56,22 @@ def test_create_run(client):
 def test_create_run_with_agent_runs(client):
     run_plan_id = "6076d1e3a216ff15b6e95e9d"
     run_configuration_id = "6326d1e3a216ff15b6e95e9d"
-    client_agent_id = "6076d152b28b871d6bdb604f"
-    client_agent_hostname = "client_masestro.net"
-    client_agent_ip = "127.0.0.4"
-    server_agent_ids = ["6076d1bfb28b871d6bdb6095"]
-    server_agent_hostname = "server1.net"
-    server_agent_ip = "127.0.0.5"
+    agent_ids = ["6076d1bfb28b871d6bdb6095", "6076d1bfb28b871d6bdb6096"]
+    agent_hostname = "agent1.net"
+    agent_hostname2 = "agent2.net"
+    agent_ip = "127.0.0.5"
+    agent_ip2 = "127.0.0.6"
```

Comment:
Creating a different data structure for agents will let us use for to iterate on the assertion part. 
suggestion:
`agents = [
    {
        "id": "6076d1bfb28b871d6bdb6095",
        "hostname": "agent1.net",
        "ip": "127.0.0.5"
    },
    {
        "id": "6076d1bfb28b871d6bdb6096",
        "hostname": "agent2.net",
        "ip": "127.0.0.6"
    }
]`

Reply:
Thanks! Changed the way how the data was inserted. 
I didn't change the assert part to `for` because it will create some complexity over tests debugging if anything goes wrong. 

If we have some cases where we need a loop for assertion we should use `@pytest.mark.parametrize` decorator. But in this case, it's also not applicable.